Sources/ScipioKit/Producer/PIF/PIFCompiler.swift

```diff
@@ -115,6 +115,7 @@ struct PIFCompiler: Compiler {
             toolchain: toolchain,
             destinationTriple: toolchain.triple,
             flags: .init(),
+            enableParseableModuleInterfaces: true,
```

Comment:
This value is not used in Scipio. So it doesn't affect anything.
But `true` is the right state.

This option means whether emitting swift modules or not.